store-and-retrieve-data/use-storagehub-sdk/end-to-end-storage-workflow.md

```diff
@@ -256,6 +302,58 @@ After a successful file upload the logs should look something like:
     await run();
     ```
 
+## Wait for Backend to Have File
+
+In this step you wire in two small helper methods:
+
+1. **`waitForMSPConfirmOnChain`**: Polls the DataHaven runtime until the MSP has confirmed the storage request on-chain. 
+2. **`waitForBackendFileReady`**: Polls the MSP backend using `mspClient.files.getFileInfo(bucketId, fileKey)` until the file metadata becomes available. Even if the file is confirmed on-chain, the backend may not yet be aware of it.
```

Comment:
Are we telling users to use these sequentially? Or just mentioning like hey there's these two helper methods? If the latter, please use an unordered list.